static/src/javascripts/projects/common/modules/commercial/geo-utils.ts

```diff
@@ -0,0 +1,31 @@
+import { getCountryCode } from '../../../../lib/geolocation';
+
+// cache the users location so we only have to look it up once
+let geo: string | undefined;
```

Comment:
I would recommend using getLocale from `@guardian/libs`, which has a nice type `CountryCode`:

```suggestion
let geo: CountryCode | undefined;
```

Reply:
Nice, thanks for letting me know! Will make a note of that 🙂